Examples/KlaviyoSwiftExamples/README.md

```
@@ -14,7 +14,7 @@ pod install
 
 ## SPM Example
 
-This [example](SPMExample) demonstrates how to integrate our SDK using SPM. Follow the steps [here](Link to swift SDK instructions) to integrate it in your app.
+This [example](SPMExample) demonstrates how to integrate our SDK using SPM. Follow the steps [here](https://github.com/klaviyo/klaviyo-swift-sdk#installation) to integrate it in your app.
```

markdownlint-cli2 (0.23.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)